docs/source/topics/processes/include/snippets/serialize/workchain_serialize.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 from aiida.engine import WorkChain
-from aiida.orm.nodes.data import to_aiida_type
+from aiida.orm import to_aiida_type
 # The basic types need to be loaded such that they are registered with
 # the 'to_aiida_type' function.
 from aiida.orm.nodes.data.base import *
```

Comment:
is this actually correct? won't these basic types be loaded simply by importing from `aiida.orm`. I've personally never had an issue not using this line

Reply:
Needs checking, it's quite possible this is outdated.

Reply:
yeh I've checked and you definitely don't, since they are all imported via the `__all__` mechanism